webpack.config.js

```diff
@@ -5,6 +5,11 @@ var definePlugin = new webpack.DefinePlugin({
     __DEBUG__: JSON.stringify(process.env.NODE_ENV === 'development'),
 });
 
+var providePlugin = new webpack.ProvidePlugin({
```

Comment:
So, as I didn't know, I went for the status quo for this PR (already big and painful, I may say! ;-) )